frontend/scripts/react-components/chart/axis-label/axis-label-component.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const percentageSuffix = (text, suffix) => `${suffix} ${text}`;
+const regularSuffix = (text, suffix) => `${text} [${suffix}]`;
+
+const AxisLabel = props => {
```

Comment:
I think this should live along with the rest of the dashboard-widget components, the implementation is coupled to what the dashboard widget endpoint returns.